client/state/ui/notices/actions.js

```diff
@@ -0,0 +1,53 @@
+/**
```

Comment:
I don't quite like UI as a catch all. There's also the fact that we'll be showing more than one notice at a time, but probably not all, so there's not even a 1-to-1 correlation between the state and what we display.